azure/service-bus-namespace/variables.tf

```
@@ -66,4 +66,10 @@ variable tags {
   type        = any
   description = "(Optional) A mapping of tags to assign to the resource."
   default     = {}
-}
\ No newline at end of file
+}
+
+variable log_analytics_workspace_id {
+  type = string
+  description = "(Required) The id of the Log Analytics Workspace where the SQL DB will log events (e.g. audit events)"
```

I suppose this variable is optional, due to the default value
```suggestion
  description = "(Optional) The id of the Log Analytics Workspace where the SQL DB will log events (e.g. audit events)"
```